.github/workflows/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+test.md/
```

Comment:
- [ ] Kindly remove this file from the `workflows` folder, one `.gitignore` file is enough in the repository 👍